packages/tokens-studio-for-figma/src/plugin/setValuesOnVariable.test.ts

```diff
@@ -38,7 +38,7 @@ describe('SetValuesOnVariable', () => {
     expect(mockSetValueForMode).toBeCalledWith(mode, 8);
   });
 
-  it('when there is no variable which is connected to the token, we create new variable', () => {
+  it('when there is no variable which is connected to the token, we create a new variable', () => {
```

Comment:
```suggestion
  it('should create a new variable when there is no variable which is connected to the token', () => {
```

Nitpicky but it tends to read better with a `should...`